### ISSUE TITLE
[Sidekick] Edit and Preview are broken from blog.adobe.com pages without the /publish segment

### ISSUE THREAD

**Issue by kptdobe** [adobe/theblog#581]
See https://github.com/adobe/helix-content-proxy/issues/279.

The reason is the missing `/publish` segment in the url. The Sidekick should add the segment if the url is accessed without.

cc @rofe

**Comment by kptdobe** [comment]
We have a lot of links (excel spreadsheets) where `/publish` is missing.

**Comment by rofe** [comment]
Now that we finally turned off `/publish` suppression on the outer CDN and use the canonical URL everywhere, we were able to remove all of the overlays in `/tools/sidekick/plugins.js` and use the out of the box sidekick features. While older outer CDN URLs without `/publish` will stick around, they will disappear eventually, as publishers use and communicate the “predicted url” as copied from the sidekick, which now returns the url with `/publish`.

I don't think we should continue to build special tooling to support both. 

cc @davidnuescheler
